.github/workflows/build.yml

```diff
@@ -0,0 +1,84 @@
+name: "Build and test Malwarecage"
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+jobs:
+  build_core:
+    name: Build Malwarecage core image
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: 1
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Build and push Malwarecage image
+        uses: docker/build-push-action@v1.1.0
```

Comment:
Docker Hub can itself add a hook to the GH repository so it would build an image by itself (like an external CI, but it's not CI, only the Docker Image builder&publisher). Maybe we could leverage this feature? We will not have to store any secrets in the config then.